ULWGL-database.csv

```diff
@@ -587,3 +587,4730 @@ Soldier of Fortune II: Double Helix - Gold Edition,gog,1228964594,ulwgl-12289645
+Spellcaster University,gog,1178128112,ulwgl-895620,,
+Frozen Cortex Game,gog,1439551206,ulwgl-237350,,
+Hand of Doom,gog,1286131410,ulwgl-1924400,,
+Convoy,gog,1432538826,ulwgl-318230,,
+Psi 5 Trading Company,gog,1719422658,ulwgl-1357550,,
+Disco Elysium - The Final Cut,gog,1771589310,ulwgl-632470,,
+SteamWorld Quest: Hand of Gilgamech,gog,1941294775,ulwgl-804010,,
+Ultimate Anime Jigsaw Puzzle,gog,1676168876,ulwgl-2319370,,
+Dear Esther: Landmark Edition,gog,1305464490,ulwgl-520720,,
+Verne: The Shape of Fantasy,gog,1291374242,ulwgl-1424600,,
+Necrodome,gog,1204965015,ulwgl-2464680,,
+The Town of Light,gog,1655492231,ulwgl-433100,,
+Deadly Dozen,gog,1956859522,ulwgl-514080,,
+Tcheco in the Castle of Lucio,gog,1572982401,ulwgl-377600,,
+Wetlands,gog,1465732730,ulwgl-1367160,,
+Scars of Summer,gog,1384959600,ulwgl-1405470,,
+Siege Survival: Gloria Victis,gog,1991069301,ulwgl-1362420,,
+Galactic Civilizations II: Ultimate Edition,gog,1444400746,ulwgl-202200,,
+Disney Classic Games: Aladdin and The Lion King,gog,1515245162,ulwgl-1126190,,
+World Turtles,gog,1759465050,ulwgl-1512050,,
+Magna Fortuna,gog,1646415448,ulwgl-1122730,,
+Saya no Uta ~ The Song of Saya Director's Cut,gog,1541066319,ulwgl-702050,,
+Bad Dream: Fever,gog,1297501772,ulwgl-769970,,
+Oniken: Unstoppable Edition,gog,1291537992,ulwgl-252010,,
+Ruiner,gog,1637928515,ulwgl-464060,,
+Civitatem,gog,1086017093,ulwgl-670820,,
+Claire's Quest: GOLD,gog,1973407267,ulwgl-1418760,,
+A Juggler's Tale,gog,1268895261,ulwgl-1252830,,
+Nongunz: Doppelganger Edition Demo,gog,1563414043,ulwgl-1580280,,
+Dark Devotion,gog,1487359513,ulwgl-718590,,
+Frozen Synapse,gog,1438076423,ulwgl-98200,,
+Mind Scanners,gog,1345277440,ulwgl-1389550,,
+Daydream: Forgotten Sorrow Demo,gog,1829195264,ulwgl-1542110,,
+LumbearJack,gog,1598934524,ulwgl-1582290,,
+Café Stella and the Reaper's Butterflies,gog,1944735225,ulwgl-1829980,,
+Star Hammer: The Vanguard Prophecy,gog,1642319334,ulwgl-337680,,
+My Memory of Us,gog,1752911333,ulwgl-651500,,
+ITTA,gog,1824443868,ulwgl-775580,,
+Travellers Rest,gog,1353960921,ulwgl-1139980,,
+Feist,gog,1437060567,ulwgl-327060,,
+Tales from Candleforth,gog,1827130835,ulwgl-2200410,,
+Tandem: A Tale of Shadows Demo,gog,1205521867,ulwgl-1436920,,
+7 Girls War,gog,1862815164,ulwgl-1279270,,
+FATE: The Cursed King,gog,1971113392,ulwgl-303690,,
+The Dead End,gog,1842662827,ulwgl-1342060,,
+Ziggurat 2,gog,1427459497,ulwgl-1159560,,
+The Book of Bondmaids,gog,1635274144,ulwgl-1398070,,
+Bite the Bullet,gog,1213517212,ulwgl-1075550,,
+Defender's Quest 2: Mists of Ruin,gog,1988808085,ulwgl-252190,,
+Hard West,gog,1442827661,ulwgl-307670,,
+The Textorcist: The Story of Ray Bibbia Demo,gog,1093291391,ulwgl-940680,,
+Cloud Gardens,gog,1107152216,ulwgl-1372320,,
+Night in the Woods: Weird Autumn Edition,gog,1330400591,ulwgl-481510,,
+B.I.O.T.A.,gog,1321782603,ulwgl-1640320,,
+The Legend of Heroes: Trails in the Sky SC,gog,1444826419,ulwgl-251290,,
```

Comment:
Looks like my ID from https://github.com/Open-Wine-Components/ULWGL-database/pull/18 is wrong. No idea where I got that number. So you can remove line 583.